FordLabs/PeopleMover
# 216 api app token support

## Issue
Resolves 216 support for app tokens

## What was done
- [x] Get labs-tools app token to break apart and find identifier
- [x] Use discovered identifier to determine if app has access to space

## How to test
1. get app token and put app id's guid into database for a particular space (user_space_mapping table)
2. using postman, make a call to the backend using the app token
3. postman call should return successfully for the space specified in step 1
